clinica/iotools/converters/aibl_to_bids/utils/clinical.py

```diff
@@ -148,17 +149,56 @@ def _load_specifications(
     return pd.read_csv(specifications, sep="\t")
 
 
+def _mapping_diagnosis(diagnosis: int) -> str:
+    if diagnosis == 1:
+        return "CN"
+    elif diagnosis == 2:
+        return "MCI"
+    elif diagnosis == 3:
+        return "AD"
+    else:
+        return "n/a"
+
+
+def _extract_metadata_df(
```

Comment:
I'd vote for a more informative name like `_build_sessions_for_rid` or something like that. WDYT ?

Reply:
Thanks 🙏